zds/tutorialv2/views/contents.py

```diff
@@ -2066,6 +2045,30 @@ def form_invalid(self, form):
         return super(RemoveAuthorFromContent, self).form_valid(form)
 
 
+class ChangeHelp(LoggedWithReadWriteHability, SingleContentFormViewMixin):
```

Comment:
Autre remarque : si JS est désactivé, le lien ne fonctionne pas car il attend une requête POST (et car il ne fait de toute façon jamais de redirection pour gérer ce cas), ce qui fait que sans JS on se prend une erreur 500.

Si on s'en fout de gérer le cas où JS est désactivé (cf. discussion de la dernière réunion de dév) mais qu'on veut tout de même avoir un retour un minimum propre, on pourrait faire : 

```suggestion
@method_decorator(require_POST, name='dispatch')
class ChangeHelp(LoggedWithReadWriteHability, SingleContentFormViewMixin):
```

Cette suggestion nécessite d'ajouter l'import correspondant : 

```py
from django.views.decorators.http import require_POST
```

Reply:
Je pense avoir trouvé encore mieux.

Reply:
Effectivement, passer par un vrai formulaire en secondaire c'est d'autant mieux, +1